### ISSUE TITLE
Using return in Route callbacks does not display value

### ISSUE THREAD

**Issue by alejoloe007jb** [skipperbent/simple-php-router#257]
not work:
```php
use Pecee\SimpleRouter\SimpleRouter as Route;
Route::get('/', function () {
return 'Hola';
});
```
work:
```php
use Pecee\SimpleRouter\SimpleRouter as Route;
Route::get('/', function () {
echo 'Hola';
});
```

**Comment by skipperbent** [comment]
Hi @alejoloe007jb .

The two examples are identical. You can find a unit test for the path `/` in the `RouterUrlTest` class which passes fine, so you might have to be a little more exact on what issues you are experiencing.

/ Simon

**Comment by skipperbent** [comment]
Update: Sorry, I see what you mean - i'll look into it.

**Comment by alejoloe007jb** [comment]
Using return in Route callbacks does not display value

**Comment by skipperbent** [comment]
Hi @alejoloe007jb .

The issue has been resolved in version 3.3.5.

Thanks for taking your time to reporting the issue.

/ Simon
